test/1.4/lib/T_attributes.dml

```diff
@@ -30,6 +30,16 @@ attribute f is (double_attr) {
 //    method set(attr_value_t value) throws default { default(value); }
 }
 
+attribute g is (bool_attr) {
```

Comment:
Silly me, this doesn't follow the naming pattern. It's not alphabetical, it's abbreviational.